common/pom.xml

```diff
@@ -26,5 +26,47 @@
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-text</artifactId>
     </dependency>
+    <dependency>
+      <groupId>info.debatty</groupId>
+      <artifactId>java-string-similarity</artifactId>
+      <version>2.0.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.xerial</groupId>
+      <artifactId>sqlite-jdbc</artifactId>
+      <version>3.36.0.3</version>
+    </dependency>
+    <dependency>
+      <groupId>org.deeplearning4j</groupId>
+      <artifactId>deeplearning4j-core</artifactId>
+      <version>1.0.0-M1.1</version>
+    </dependency>
+    <dependency>
+      <groupId>org.deeplearning4j</groupId>
+      <artifactId>deeplearning4j-nlp</artifactId>
+      <version>1.0.0-M1.1</version>
+    </dependency>
+    <dependency>
+      <groupId>edu.mit</groupId>
+      <artifactId>jwi</artifactId>
+      <version>2.2.3</version>
+    </dependency>
+    <dependency>
+      <groupId>com.github.DonatoMeoli</groupId>
+      <artifactId>WS4J</artifactId>
+      <version>master-c3a97d944b-1</version>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
+      <scope>test</scope>
+    </dependency>
   </dependencies>
+
+  <repositories>
```

Comment:
Maven Central does not like other repositories , we need to look at the dependencies here .

Reply:
I can give some insight about the added dependencies:

- `info.debatty:java-string-similarity` is no longer used and can be removed
- `org.xerial:sqlite-jdbc` is used to retrieve word vectors from sqlite databases (GloVe, Nasari)
- `org.deeplearning4j:deeplearning4j-nlp` is used by the fastText measure to load word vectors
- `edu.mit:jwi` is used to load the wordnet dictionary
- `com.github.DonatoMeoli:WS4J` provides implementations for various wordnet algorithms
  - this dependency is the reason jitpack was added as a repository
- `org.junit.jupiter:junit-jupiter-engine` used for testing

Reply:
Thanks for the information!
Since we are not merging this to main, its not that critical now :) 

> `com.github.DonatoMeoli:WS4J` provides implementations for various wordnet algorithms
> 
>     * this dependency is the reason jitpack was added as a repository

I think before merging to main, we have to think about this dependency.